### DISCUSSION TITLE
LSPosed是如何获取到姓名的？

### DISCUSSION THREAD

**Discussion by TsiriusT** [LSPosed/LSPosed#2571] [Q&A] [answered]
LSPosed是如何获取到姓名的？为什么会在在模块栏显示名字

**Comment by suqiernb** [comment, marked as answer]
LSPosed 获取的是机主的用户名

**Reply by StarsWhere** [reply]
是的，问题是这是如何获取的？在折腾手机时突然看到自己的真实姓名可不是什么美妙的感觉。如果你知道，能不能告知我们相关的原因或者给予我们讲解的链接？

**Reply by yujincheng08** [reply]
use any terminal, type

su
pm list users
